docs/organize.md

```
@@ -1 +1,144 @@
+Currently, there are four types of issue templates
+available for use. You can find the code for these
+templates in the `.github/ISSUE_TEMPLATE` folder.
+
+If needed, you can extend or remove any templates.
+Out of the box, issues created with one of the
+templates will be automatically labeled.
+
+Please check the official GitHub documentation for
+further information.
+
+**Note**: Issue forms are currently a preview 
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)